src/google/cloud/ndb/tasklets.py

```diff
@@ -14,8 +14,44 @@
 
 """Provides a tasklet decorator and related helpers.
 
-Tasklets are a way to write concurrently running functions without
-threads.
+Tasklets are a way to write concurrently running functions without threads.
+Tasklets are executed by an event loop and can suspend themselves blocking for
+I/O or some other operation using a yield statement. The notion of a blocking
+operation is abstracted into the Future class, but a tasklet may also yield an
+RPC in order to wait for that RPC to complete.
+
+The @tasklet decorator wraps generator function so that when it is called, a
+Future is returned while the generator is executed by the event loop. Within
+the tasklet, any yield of a Future waits for and returns the Future's result.
+For example::
+
+    @tasklet
+    def foo():
+        a = yield <some Future>
+        b = yield <another Future>
+        raise Return(a + b)
```

Comment:
Just: return a + b

Raising `Return` is still supported, but we shouldn't suggest it as the way to do it and should be considered deprecated. I believe the need for "raise Return" is from an age when Python didn't yet support returning from a generator.